### ISSUE TITLE
I have to toggle "filter content" every time

### ISSUE THREAD

**Issue by simplenotezy** [Sequel-Ace/Sequel-Ace#378]
- Sequel Ace Version: 2.1.8

**Description**
I need to go to Table -> Filter Content if I wish to filter a table's content. Usually it would remember my setting, and also by default show it.

**Steps To Reproduce**
1. Go to Content
2. Toggle Table -> Filter Content
3. Change database / or quit Sequel Ace and open it again
4. You have to redo step number 2

**Expected Behaviour**
Step 3 shouldn't be needed.

**Comment by Jason-Morcos** [comment]
I don't believe that the behavior you're describing as desired is typical macOS behavior. I like to think of filtering a little like Safari's find in page. If you go to a different page or quit safari, the find box goes away until you toggle it back. Once you do toggle it back, however, it stays up until you quit the app or go to a different page.

Am I misinterpreting your request? It seems you'd like the filter bar to reappear if you quit and reopen the app if you were filtering before you quit it? This is not a bug, in my opinion, but is just typical macOS behavior.

**Comment by simplenotezy** [comment]
@Jason-Morcos Have a look at this issue: https://github.com/sequelpro/sequelpro/issues/3084 I would simply like the filters to appear by default, because I use it 24/7 when inspecting content, and also there is no keybinding for it. So I find my self having to do a lot of useless clicks to get it out, every single time I view data 😊

**Comment by Jason-Morcos** [comment]
> @Jason-Morcos Have a look at this issue: https://github.com/sequelpro/sequelpro/issues/3084 I would simply like the filters to appear by default, because I use it 24/7 when inspecting content, and also there is no keybinding for it. So I find my self having to do a lot of useless clicks to get it out, every single time I view data 😊 

Alrighty. Well there is a key binding to show the filters - Cmd+f shows them! It's what I use to show them when needed

**Comment by simplenotezy** [comment]
That's relevant @Jason-Morcos - perhaps it should be visible in the topmenu under "Table -> Filter Content".
However, I still think it would be nice, if there was a setting to always have this visible, or perhaps just remember the state.

**Comment by kjc75** [comment]
I know when I'm in a database table view, the thing I'm almost certainly doing is going to require a filter to find the record I'm after.  While the Query editor would be the "proper" place, having at least an option for that first filter field to show by default would be ideal.  Can consider it the equivalent to the Mail.app and Messages.app search field, but with more explicitly visible scoping.

**Comment by jamesstout** [comment]
Hmm, I can't reproduce this. The filter field is always visible when I go to content...

**Comment by simplenotezy** [comment]
@jamesstout what version? Take a screenshot of the "about Sequel Ace"

**Comment by RickTuckerUSA** [comment]
![2021-01-25 Sequel Ace - multiple filters](https://user-images.githubusercontent.com/25371243/105724357-c597f200-5ef5-11eb-9235-a74f013bee4d.jpg)

I enjoyed and constantly (meaning all the time) used a feature of Sequel Ace that seems to have gone away. For a particular table (in this case, Games), I add multiple filters (as shown in the screen image) and turn some on and off during my work in cataloguing game information. A few weeks ago, I could switch tables (e.g. to the GamePublisher table or to another table) and then return to the Game table and all of my filters would still be there. However, recently, every time I switch between tables, all established filters disappear and I have to enter the filters again and again, in every table I open. It would be great to have the "sticky" filters return. If you need more explanation, let me know.

Thanks, and all the best.

**Comment by Jason-Morcos** [comment]
> ![2021-01-25 Sequel Ace - multiple filters](https://user-images.githubusercontent.com/25371243/105724357-c597f200-5ef5-11eb-9235-a74f013bee4d.jpg)
> 
> 
> 
> I enjoyed and constantly (meaning all the time) used a feature of Sequel Ace that seems to have gone away. For a particular table (in this case, Game), I add multiple filters (as shown in the screen image) and turn some on and off during my work in cataloguing game information. A few weeks ago, I could switch tables (e.g. to the GamePublisher table or to another table) and then return to the Game table and all of my filters would still be there. However, recently, every time I switch between tables, all established filters disappear and I have to enter the filters again and again, in every table I open. It would be great to have the "sticky" filters return. If you need more explanation, let me know.
> 
> 
> 
> Thanks, and all the best.

We believe this issue was fixed in the latest 3.1.1 release. Are you using 3.1.1?

**Comment by RickTuckerUSA** [comment]
> > ![2021-01-25 Sequel Ace - multiple filters](https://user-images.githubusercontent.com/25371243/105724357-c597f200-5ef5-11eb-9235-a74f013bee4d.jpg)
> > I enjoyed and constantly (meaning all the time) used a feature of Sequel Ace that seems to have gone away. For a particular table (in this case, Game), I add multiple filters (as shown in the screen image) and turn some on and off during my work in cataloguing game information. A few weeks ago, I could switch tables (e.g. to the GamePublisher table or to another table) and then return to the Game table and all of my filters would still be there. However, recently, every time I switch between tables, all established filters disappear and I have to enter the filters again and again, in every table I open. It would be great to have the "sticky" filters return. If you need more explanation, let me know.
> > Thanks, and all the best.
> 
> We believe this issue was fixed in the latest 3.1.1 release. Are you using 3.1.1?

Excellent! I had checked for a new version last week, and indeed 3.1.1 has the filters working again. Thanks to all!
